packages/expo-modules-autolinking/scripts/ios/cocoapods/installer.rb

```diff
@@ -0,0 +1,76 @@
+# Overrides CocoaPods `Installer`/'Podfile' classes to patch podspecs on the fly
+# See: https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer.rb
+# See: https://github.com/CocoaPods/Core/blob/master/lib/cocoapods-core/podfile.rb#L160
+#
+# This is necessary to disable `USE_FRAMEWORKS` for specific pods that include
+# React Native Core headers in their public headers, which causes issues when
+# building them as dynamic frameworks with modular headers enabled.
+
+module Pod
+  class Podfile
+    public
+
+    def framework_modules_to_patch
+      @framework_modules_to_patch ||= ['ExpoModulesCore', 'Expo', 'ReactAppDependencyProvider', 'expo-dev-menu']
+    end
+
+    def expo_add_modules_to_patch(modules)
+      framework_modules_to_patch.concat(modules)
+      Pod::UI.puts "[Expo] ".blue + "Added extra framework modules to patch: #{modules.join(', ')}"
```

Comment:
```suggestion
```

the following log will show all `framework_modules_to_patch` which is good enough